### PR TITLE
Added cross validation on feature selection and trained final model - For issue #20

### PR DESCRIPTION
**Pull request by RobBlumberg**
I transformed the original training set to order=5, and then ran cross validation on the number of features to select. Then I trained a linear model with the optimized hyperparameters and got results on a test set (this test set is a subset of training_data, not the actual test_data).

### CODE REVIEW

**Review by v5y8 — APPROVED** [review]
looks good. merging
